lib/Models/Catalog/Ows/WebFeatureServiceCatalogItem.ts

```diff
@@ -359,8 +387,8 @@ class WebFeatureServiceCatalogItem extends GetCapabilitiesMixin(
             request: "GetFeature",
             typeName: this.typeNames,
             version: "1.1.0",
-            outputFormat: supportsGeojson ? "JSON" : "gml3",
-            srsName: "urn:ogc:def:crs:EPSG::4326", // srsName must be formatted like this for correct lat/long order  >:(
+            outputFormat: supportsGeojson ? "JSON" : this.outputFormat, // Will choose best option from GetCapabilities response TODO: should we move JSON conditional stuff to where default is set too?
```

Comment:
Following from previous comment

```suggestion
         outputFormat: this.outputFormat,
```